libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/components/avatar/AvatarShape.kt

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2025 New Vector Ltd.
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only OR LicenseRef-Element-Commercial
+ * Please see LICENSE files in the repository root for full details.
+ */
+
+package io.element.android.libraries.designsystem.components.avatar
+
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Shape
+
+@Composable
+fun avatarShape(
```

Comment:
Maybe this could be `AvatarType.avatarShape()` instead? That way it won't appear by default in the method autocompletion.

Reply:
Better, thanks! https://github.com/element-hq/element-x-android/pull/4921/commits/2bcffe80de57031f34a3f076203fac8be42dab6b